### PR TITLE
Ds 139 pokeball spinner positioning

### LINKED ISSUE #139: Pokeball spinner logo positioning
**Is your feature request related to a problem? Please describe.**
Pokeball spinner positioning is incorrect while a user scrolls down under "All Pokemon" tab.
The spinner stays on the top of the page.
The spinner should move down, as the user scrolls.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Reposition spinner logo on main page
2. Add additionaly logo styling to background
3. Remove loader for modal operations
4. Add "Loading..." message to modal operations
5. Delete unnessecary existing classes and <Spinner /> imports
Closes #139

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Looks good.

File: src/components/BaseStats.js [hunk 1]
```diff
@@ -83,7 +82,9 @@ const BaseStats = ({ stats, types }) => {
             }}
           />
         ) : (
-          <Spinner className="modalSpinner" />
+          <h3 style={{
+            margin: "50px auto"
+          }}>Loading...</h3>
```
Comment (BlackCubes) [inline src/components/BaseStats.js:87]: I think you'll like styled-components. It's the same thing, but cleaner 😘

**Review by BlackCubes — APPROVED** [review]
Looking good!

**Review by Lap343 — APPROVED** [review]
Nice job! Code looks good

**Pull request merged**